block_my_feedback.php

```diff
@@ -73,23 +73,27 @@ public function get_content() : stdClass {
      *  Get my feedback call.
      *
      * @return array feedback items.
+     * @throws coding_exception
+     * @throws dml_exception
+     * @throws moodle_exception
      */
     public function fetch_feedback() : array {
-        global $CFG, $DB, $USER, $OUTPUT;
+        global $DB, $USER;
+
         // Return users 5 most recent feedbacks.
 
         // Limit to last 3 months.
         $since = time() - (2160 * 3600); // 2160 = 90 days from today as hours.
```

Comment:
I think this can use $since = strtotime('-3 month'); to be more readable.

Reply:
Cheers, have used that now.